vividus-plugin-datetime/src/main/java/org/vividus/expression/ShiftDateExpressionProcessor.java

```diff
@@ -57,19 +55,7 @@ protected String evaluateExpression(Matcher expressionMatcher)
                 format);
         DateExpression dateExpression = new DateExpression(expressionMatcher, MINUS_SIGN_GROUP, PERIOD_GROUP,
                 DURATION_GROUP, FORMAT_GROUP);
-        if (dateExpression.hasPeriod())
-        {
-            zonedDateTime = dateExpression.processPeriod(zonedDateTime);
-        }
-        if (dateExpression.hasDuration())
-        {
-            zonedDateTime = dateExpression.processDuration(zonedDateTime);
-        }
-        return format.format(zonedDateTime);
-    }
-
-    private String normalize(String argument)
-    {
-        return StringUtils.replace(argument.trim(), "\\,", ",");
+        String formattedDateTime = dateExpression.format(zonedDateTime, Locale.ENGLISH);
+        return normalize(formattedDateTime);
```

Comment:
why is result normalized?

Reply:
my mistake. Previously was used  `DateTimeFormatter format = DateTimeFormatter.ofPattern(normalize(expressionMatcher.group(FORMAT_GROUP)),` which is normalize, but as logic for formatiing moved to the `DateExpression` it is also should be normalized inside it. fixed